shared/activate.ts

```diff
@@ -98,6 +102,34 @@ export async function activateCodeIntel(
     }
 }
 
+export async function activateLsifUnderlines(context: sourcegraph.ExtensionContext): Promise<any> {
```

Comment:
Would be good to talk with the @sourcegraph/web team to see if there is an easy path forward with the extension host so they can control the highlighting instead of changing the cssText of elements directly.

Reply:
I force pushed this in the middle of a call with @AlicjaSuska to discuss how to move forward with this prototype. This was not at all ready for review yet 😅

Reply:
😛